demo/pymediasoup-client/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM python:3.10-slim
+
+# Install system dependencies 
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    libssl-dev \
+    libffi-dev \
+    python3-dev \
+    ffmpeg\
+    && rm -rf /var/lib/apt/lists/*
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY /src /app
+
+COPY /patch /patch
```

Comment:
Now that pymediasoup 1.0.1 has been released, we need to remove the patch (and pypatch from requirements)

Reply:
Solved! Remove patch into Dockerfile